src/components/dev/LocalizationReport.js

```diff
@@ -30,31 +40,25 @@ const ReportItem = function ({ keyPath, test, locKey, pass }) {
 };
 
 
-const ReportListItems = function ({ keyPath, test, locKey, pass }) {
-  return (
-    <ReportItem
-      key     = { keyPath }
-      keyPath = { keyPath }
-      test    = { test }
-      locKey  = { locKey }
-      pass    = { pass } />
-  );
-};
+const ReportList = function ({ results }) {
 
+  let items = [];
+  for (let result of results) {
```

Comment:
I would like to add `.map()`(/`.reduce()`) to the discussion topics for coding conventions :)